lib/matplotlib/tests/test_cycles.py

```diff
@@ -129,7 +129,7 @@ def test_property_collision_plot():
     fig, ax = plt.subplots()
     ax.set_prop_cycle('linewidth', [2, 4])
     for c in range(1, 4):
-        ax.plot(np.arange(10), c * np.arange(10), lw=0.1)
+        ax.plot(np.arange(10), c * np.arange(10), lw=0.1, color='k')
```

Comment:
This still tests that, just have to include all of the keys in the cycler
and as implemented on v2.x and master, 'color' is always in the cycler

On Fri, May 6, 2016, 16:40 Benjamin Root <notifications@github.com> wrote:

> In lib/matplotlib/tests/test_cycles.py
> <https://github.com/matplotlib/matplotlib/pull/6374#discussion_r62388023>:
>
> > @@ -129,7 +129,7 @@ def test_property_collision_plot():
> >      fig, ax = plt.subplots()
> >      ax.set_prop_cycle('linewidth', [2, 4])
> >      for c in range(1, 4):
> > -        ax.plot(np.arange(10), c * np.arange(10), lw=0.1)
> > +        ax.plot(np.arange(10), c * np.arange(10), lw=0.1, color='k')
>
> This unit test is explicitly testing that the property cycler does not
> advance if the user specifies the property provided by the cycler. That was
> the original intent. The fact that it came about because of a slightly
> different bug doesn't matter.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/6374/files/faae5069bb987fd5447652860a8d1fb602c3289d#r62388023>
>